geode-core/src/main/java/org/apache/geode/cache/query/internal/index/PartitionedIndex.java

```diff
@@ -58,6 +58,10 @@
   private Map<Region, List<Index>> bucketIndexes =
       Collections.synchronizedMap(new HashMap<Region, List<Index>>());
 
+  // An arbitrary bucket index from this PartiionedIndex that is used as a representative
```

Comment:
Spelling. `PartitiionedIndex` => `PartitionedIndex`

Reply:
This should now be fixed.  Eagle eye ;-)